UnitySDK/ProjectSettings/TagManager.asset

```diff
@@ -26,6 +26,14 @@ TagManager:
   - pyramid
   - switchOn
   - stone
+  - walkableSurface
+  - target
+  - symbol_O
```

Comment:
i'll clean this up as much as possible. hard to tell what tags are in use since there's no built in methods to do this. may be prudent to do tag clean up in a new PR in case a random script is using a tag that gets removed

Reply:
I actually think we should be careful not to increase complexity in this PR. I would strongly encourage removing unused tags before merging.